nuxeo-features/nuxeo-automation/nuxeo-automation-test/src/test/java/org/nuxeo/ecm/automation/server/test/EmbeddedAutomationClientTest.java

```diff
@@ -1089,4 +1090,19 @@ public void testContextInjection() throws IOException {
         assertEquals("hello", folder.getString("dc:title"));
         assertNull(folder.getString("dc:description"));
     }
+
+    @Test
+    @LocalDeploy("org.nuxeo.ecm.automation.test.test:test-no-stack-trace-properties.xml")
+    public void shouldNotHaveTheRemoteCause() {
+        try {
+            // get a wrong doc
+            Document wrongDoc = (Document) session.newRequest(FetchDocument.ID).set("value", "/test").execute();
+            fail("Unexpected " + wrongDoc);
+        } catch (RemoteException e) {
+            assertNotNull(e);
+            assertNull(e.getRemoteCause());
+        } catch (Exception e) {
+            fail();
```

Comment:
You can remove the `catch (Exception e) { fail(); }`, it'll give a clearer error when failing.